MoEPluginGeofence/MoEGeofenceBridge.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MoEGeofenceBridge.swift
+//  MoEngageGeofencePlugin
+//
+//  Created by Rakshitha on 23/06/22.
+//
+
+import Foundation
+import MoEngageGeofence
+import MoEPluginBase
+
+
+@objc public class MoEGeofenceBridge: NSObject {
```

Comment:
@RakshithaAcharya Please check if protocol would suffice here instead of singleton

Reply:
@deepa-iOS-SDK  This class is called from hybrid class. Since every module follows the same structure, I have implemented the same. It helps in scaling in future